README.md

```diff
@@ -601,6 +602,14 @@ The *GPG VM* does not have network access, and the Qubes split-gpg mechanism res
 * An adversary can store and view any message that is being decrypted by the *SecureDrop Workstation*.
 * An adversary can attempt to elevate their privileges and escape the VM.
 
+#### What Compromise of the *Log VM* (`sd-log`) Can Achieve
```

Comment:
Thanks for updating the threat model docs, one other aspect I thought of is that code execution/compromise of any VM using this central logging facility can exhaust sd-log vm storage.

Reply:
good point, added a note about this